src/routes/login.js

```diff
@@ -0,0 +1,27 @@
+const router = require('express').Router();
+const User = require("../models/user.model");
+
+
+/**
+ * Login
+ */
+router.route('/').post((req, res) => {
```

Comment:
in terms of design, do we want to separate the concerns of receiving data on a endpoint and accessing the database. We might want to have one controller, whose responsibility is to receive data and dispatch to the right class.
https://www.geeksforgeeks.org/front-controller-design-pattern/
We should put the logic that encapsulate how we access the data sources in an other class.
https://learn.microsoft.com/en-us/dotnet/architecture/microservices/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design#the-repository-pattern

Reply:
Router.route should go in the controller.

User.findOne should go in the repository.